### ISSUE TITLE
Set aria-live on input box in quickpick so that errors are announced in real time. 

### ISSUE THREAD

**Issue by neeleshb** [microsoft/vscode#145771]
It would be very useful to have the input box in quickinput announce the errors (e.g. missing pattern etc) in real time. setting aria-live on quickinput can fix this.

**Comment by isidorn** [comment]
The fix for this issue resulted in a regression https://github.com/microsoft/vscode/issues/146636
Basically all content in quick pick was double read. Thus I had to revert this. So removing this issue from the milestone since it was not actually tackled. 
Neelesh if you have an alternative solution to this problem please let us know. Thanks
